vita3k/renderer/src/scene.cpp

```diff
@@ -108,9 +115,14 @@ COMMAND(handle_sync_surface_data) {
         // unlocking before a notify should be faster
         lock.unlock();
         renderer.notification_ready.notify_all();
-    }
+    };
+
+    if (renderer.disable_surface_sync)
+        // do it as soon as possible
+        signal_notifications();
 
     if (renderer.current_backend == Backend::Vulkan) {
+        ;
```

Comment:
?

Reply:
Yes i have already signaled is just forget